semgrep/semgrep/semgrep_main.py

```diff
@@ -160,6 +160,7 @@ def main(
     no_git_ignore: bool = False,
     timeout: int = DEFAULT_TIMEOUT,
     max_memory: int = 0,
+    max_target_size: int = 0,
```

Comment:
Suggestion:

```suggestion
    max_target_bytes: int = 0,
```

Adding units to the variable name removes need for documentation

Reply:
nice, let's do that